README.md

```diff
@@ -2,7 +2,7 @@
 
 # Budget Manager
 
-This is a simple appliction built with ruby on rails to manage your budget, designed for the mobile screens.
+Budget manager is a simple and useful application that can help you to manage your budget. You can create a new account and begin by adding some categories that has a meaning to you. You'll than be able to add new transactions to keep track of your expenses and income in a simple and organized way.
 
```

Comment:
- [ ] Kindly add a link to your online version in your readme file, it's a project requirement.